app/src/main/jni/moonlight-core/simplejni.c

```diff
@@ -20,7 +20,7 @@ Java_com_limelight_nvstream_jni_MoonBridge_sendMousePosition(JNIEnv *env, jclass
 JNIEXPORT void JNICALL
 Java_com_limelight_nvstream_jni_MoonBridge_sendMouseMoveAsMousePosition(JNIEnv *env, jclass clazz,
         jshort deltaX, jshort deltaY, jshort referenceWidth, jshort referenceHeight) {
-    LiSendMouseMoveAsMousePositionEvent(deltaX, deltaY, referenceWidth, referenceHeight);
+    LiSendMousePositionEvent(deltaX, deltaY, referenceWidth, referenceHeight);
```

Comment:
It looks like you somehow partially reverted b3700b5a19a8f2a6df1d4a56b1e7e053054545a3. Please undo that part.